types/to-readable-stream/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for url-parse-lax 5.0
+// Project: https://github.com/sindresorhus/url-parse-lax
+// Definitions by: Woo Yong Seok <https://github.com/woo-yong0405>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { Readable } from "stream";
+
+export default function toReadableStream(input: string | Uint8Array): Readable;
```

Comment:
```ts
// Type definitions for url-parse-lax 5.0
// Project: https://github.com/sindresorhus/url-parse-lax
// Definitions by: Woo Yong Seok <https://github.com/woo-yong0405>
// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped

/// <reference types="node" />

import { Readable } from "stream";

export default function toReadableStream(input: string | Uint8Array): Readable;
```
this looks wrong, please review which module you're adding, for now it looks like it's subject is `url-parse-las` module.

Reply:
Done. Thank you for the reply.